test/cpp/jit/test_gpu.cpp

```diff
@@ -3448,6 +3448,78 @@ void testGPU_FusionBCastReduce() {
       !tv2->axis(2)->isBroadcast() && !tv2->axis(2)->isReduction());
 }
 
+void testGPU_FusionComputeAtExprOrder() {
+  {
+    torch::jit::fuser::cuda::CudaKernel prog;
+    Fusion& fusion = *prog.fusion_;
+    FusionGuard fg(&fusion);
+
+    // Set up your input tensor views
+    TensorView* tv0 = makeDummyTensor(1);
+    fusion.addInput(tv0);
+
+    auto tv1 = add(tv0, new Float(1));
+    auto tv2 = add(tv0, new Float(1));
+    TensorView* tv3 = add(tv1, tv2);
+    fusion.addOutput(tv2);
+    fusion.addOutput(tv3);
+
+    tv1->computeAt(tv3, -1);
```

Comment:
can you also check with just `tv2->computeAt(tv3...`) in this test?

Reply:
Added.